app/jobs/add_registration_job.rb

```diff
@@ -17,9 +17,10 @@ def self.prepare_task(user_id, competition_id)
   end
 
   def perform(lane_name, competition_id, user_id, lane_params)
-    # Hotfix for mass stuck duplicate registrations in the queue
-    registration = Registration.find_by(competition_id: competition_id, user_id: user_id)
-    return if registration.present?
+    if Registration.where(competition_id: competition_id, user_id: user_id).exists?
```

Comment:
Isn't this just Registration.exists?(competition_id: competition_id, user_id: user_id)
At least it looks like it from this https://apidock.com/rails/ActiveRecord/Base/exists%3F/class

Reply:
Ah good call! Changed